css/style.css

```diff
@@ -136,12 +165,57 @@ footer {
     nav ul {
         display: none;
     }
+
+    .profile-text {
+        /*width: 100%;*/
+        float: none;
+    }
 }
 
 @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
     /* IE10+ CSS here */
 
+    .container .video-wrapper {
+        box-shadow: none;
+    }
+
     .video-wrapper video {
         height: auto;
     }
-}
\ No newline at end of file
+
+    .container .about-wrapper {
+        box-shadow: none;
+    }
+
+    .about-wrapper .scroll-icon {
+        bottom: -700px;
+    }
+}
+
+
+.section-header {
+    padding-top: 10px;
+    height: 250px;
+}
+
+.front-text {
+    color: #ffffff;
+    font-family: 'Muli', sans-serif;
+    font-weight: 700;
+    font-size: 4rem;
+    z-index: 2;
+    position: relative;
+    top: 55px;
+    text-transform: uppercase;
+}
+
+.shadow-text {
+    font-family: 'Covered By Your Grace', cursive;
+    color: #AA6373;
```

Comment:
Might not be necessary - Turning off in inspector didn't seem to make any difference